### PR TITLE
Issue 7104 update Javamail-1.6 jars to 1.6.2

### PR DESCRIPTION
**Pull request by uberskigeek**

### CODE REVIEW

**Comment by uberskigeek** [comment]
Changes were tested using the com.ibm.ws.javamail.1.6_fat fat bucket.

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

File: dev/com.ibm.websphere.javaee.mail.1.6/bnd.bnd [hunk 1]
```diff
@@ -9,11 +9,10 @@
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/bundle.props
-bVersion=1.0
+bVersion=1.1
```
Comment (aguibert) [inline dev/com.ibm.websphere.javaee.mail.1.6/bnd.bnd]: did this need to be changed?

File: dev/com.ibm.websphere.javaee.mail.1.6/build.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <property name="spec.name" value="mail"/>
     <property name="spec.marketing.version" value="1.6"/>
     <property name="spec.javadoc.name" value="javamail-1.6.0-javadocs.zip"/>
-    <property name="bundle.version" value="1.0"/>
+    <property name="bundle.version" value="1.1"/>
```
Comment (aguibert) [inline dev/com.ibm.websphere.javaee.mail.1.6/build.xml]: this is an unused file and can be deleted

File: dev/com.ibm.ws.com.sun.mail.javax.mail.1.6/bnd.overrides [hunk 3]
```diff
@@ -1,16 +1,16 @@
 -include= ~../cnf/resources/bnd/rejar.props
 
-bVersion=1.6
+bVersion=1.7
```
Comment (aguibert) [inline dev/com.ibm.ws.com.sun.mail.javax.mail.1.6/bnd.overrides]: did this need to be updated?

File: dev/com.ibm.ws.javamail.1.6/bnd.bnd [hunk 4]
```diff
@@ -10,7 +10,7 @@
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/bundle.props
 
-bVersion=1.6
+bVersion=1.7
```
Comment (aguibert) [inline dev/com.ibm.ws.javamail.1.6/bnd.bnd]: same comment

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

**Pull request merged**